src/test_l1_hdmi_cec_driver.c

```diff
@@ -596,31 +606,34 @@ void test_hdmicec_hal_l1_addLogicalAddress_sinkDevice_negative( void )
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
 
     result = HdmiCecAddLogicalAddress( handle, logicalAddress );
+#ifdef EHNCAGED_CHECKS_ENABLED
```

Comment:
why is this one different that all the others?

Reply:
This line was missed when updating.
Updated this line as well.